README.md

```diff
@@ -310,19 +311,20 @@ Note that build cleans previously created configuration. To prevent removal of c
 * Remove Splunk container - `make docker-splunk-remove`
 * Restore configuration - `make dev-config-backup`
 * Backup configuration - `make dev-config-restore` - it's not clear whether it really works
-* Tail Splunk logs - `make docker-tail-logs`
+* Tail Splunk logs - `make docker-tail-logs-f`
 
 # E2E Testing
 
 We are using Playwright - https://playwright.dev/
 
 ## Setup
 
-* Install modules - `npm install`
+* Install Node - `brew install node`
+* Install modules - `make e2e-install`
 * Create `.env` file based on [.env.example](.env.example) with access keys
 
 ## Run tests
 
 * Make sure, that your Splunk is running - `make docker-splunk-run`
-* Use `make e2e` - to run e2e tests without the browser
-* Use `make e2e-headed` - to run e2e tests with the browser
+* Use `make e2e-test` - to run e2e tests without the browser
```

Comment:
Actually, we need a browser to run the tests in any case. It would just be in the headless mode.

Reply:
Good catch! :) Now the browser is installed as part of `make e2e-install`.